src/style.css

```
@@ -1 +1,153 @@
-/* add styles here */
+/* #region @font-face */
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+/* #endregion */
+
+/* #region body */
+body {
+  margin: 0;
+
+  color: #3d4e61;
+
+  --border-for-input: 1px solid #e1e7ed;
+  --radius-for-input: 4px;
+  --width-for-input: 100%;
+  --default-box-shadows: 0 1px 8px #3d4e611a;
+  --hover-box-shadows: 0 4px 4px #00000040;
+  --focus-box-shadows: 0 4px 4px #00000040;
+  --focus-text-shadows: 0 4px 4px #00000040;
+  --focus-background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+/* #endregion */
+
+/* #region search */
+.search {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  margin-top: 20px;
+}
+
+.search__form {
+  position: relative;
+}
+
+/* #endregion */
+
+/* #region logo */
+.search_logo {
+  position: absolute;
+  display: block;
+
+  background-image: url(./images/Search.svg);
+  background-size: contain;
+  background-repeat: no-repeat;
+}
+
+.search__logo--big-form {
+  height: 19px;
+  width: 19px;
+
+  top: 25px;
+  left: 26px;
+}
```

re
```suggestion
  background-repeat: no-repeat;
  height: 19px;
  width: 19px;

  top: 25px;
  left: 26px;
}
```
redundant to set this styles for big-form modifier. set them for search_logo and change only for small form.

same for other elements